resources/ibm-credentials-iam.env

```diff
@@ -1,2 +1,4 @@
 WATSON_APIKEY=5678efgh
-WATSON_AUTH_TYPE=iam
\ No newline at end of file
+WATSON_AUTH_TYPE=iam
+WATSON_URL=https://gateway-s.watsonplatform.net/watson/api
+WATSON_DISABLE_SSL=False
```

Comment:
Add some test properties that include a multi-word service name, rather than just "WATSON".

Reply:
Erika can you add these tests properties?